doc/manual/de/config/customizing.rst

```diff
@@ -0,0 +1,273 @@
+.. _customizing:
+
+Anpassungen vornehmen
+=====================
+
+Die CometVisu kann auf viele verschiedene Weisen an die eigenen Bedürfnisse angepasst werden. Im Folgenden
+werden die verschiedenen Möglichkeiten - sortiert nach Komplexitätsgrad - im Detail beschrieben.
+
+===   =============================    ================    ===================================================
+\     Anpassung                        Veränderung          Beschreibung
+===   =============================    ================    ===================================================
+1.    Eigene CSS-Regeln                Optisch             Überschreiben der vorhandenen CSS-Regeln eines Designs durch das Laden von zusätzlichen CSS-Dateien
```

Comment:
Please stick to common line lengths (optimal: 80 chars, extended ~120 chars) to keep code and display of it (e.g. here on GitHub) clean

Reply:
That is not possible with the table syntax of rst.

Reply:
ok, this would require the complex table syntax then :(